.travis.yml

```diff
@@ -17,3 +17,6 @@ deploy:
   skip_cleanup: true
   on:
     branch: master
+
+script:
+  - npm test
```

Comment:
This isn't necessary, travis detects this as a node project and runs `npm test` by default. The style checks haven't been integrated into `npm test` yet.